src/dashboard/apps/core/models/entity.py

```diff
@@ -0,0 +1,25 @@
+"""Dashboard core app models."""
+
+from django.db import models
+
+from apps.auth.models import DashboardUser as User
+from apps.core.models.abstract import DateBase
+
+
+class Entity(DateBase):
+    """Represents an operator in the system."""
+
+    name = models.CharField(max_length=64, unique=True)
+    user = models.ManyToManyField(User)
+    is_enabled = models.BooleanField(default=True)
```

Comment:
```suggestion
    is_active = models.BooleanField(default=True)
```

Reply:
Do you think we need to inactivate entities?

Reply:
This is a concept present in the Carbure entity model

Reply:
I removed it